pkg/apis/crd/v1alpha1/eniconfig_types.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ENIConfigSpec defines the desired state of ENIConfig
+type ENIConfigSpec struct {
+	SecurityGroups []string `json:"securityGroups"`
+	Subnet         string   `json:"subnet"`
+}
+
+// ENIConfigStatus defines the observed state of ENIConfig
+type ENIConfigStatus struct {
+	// Fill me
```

Comment:
Nit: If ENIConfigStatus is not used, can we remove it?

Reply:
Yeah, just the standard `Spec` and `Status` objects for CRDs/K8S resources. Although, we're not using it right now for ENIConfig CRD, it is good to have them in place.